interface/patient_tracker/patient_tracker.php

```diff
@@ -69,7 +69,8 @@
 $appointments_status = getApptStatus($appointments);
 $lres = sqlStatement("SELECT option_id, title FROM list_options WHERE list_id = ? AND activity=1", array('apptstat'));
 while ( $lrow = sqlFetchArray ( $lres ) ) {
-    $statuses_list[$lrow['option_id']] = substr($lrow['title'],2);
+    $splitTitle = explode(' ', $lrow['title']);
+    $statuses_list[$lrow['option_id']] = $splitTitle[1];
```

Comment:
A better route to take would be:
If the 2nd character is a space, then do explode, then array_shift, then implode.
Else just use the full string.